example/App.js

```diff
@@ -131,7 +134,12 @@ export const App = () => {
     return (
       <SafeAreaView style={[backgroundStyle, {flex: 1}]}>
         <StatusBar barStyle="dark-content" />
-        <ScrollView testID="DateTimePickerScrollView">
+        <ScrollView
+          testID="DateTimePickerScrollView"
+          ref={scrollRef}
+          onContentSizeChange={() =>
+            scrollRef.current?.scrollToEnd({animated: true})
```

Comment:
I'm afraid this is breaking the e2e tests on android as some ui elements are not visible. We need to get a testing emulator with a bigger screen I guess, do you want to take a look at this?

thanks

Reply:
I will take a look on this.

Reply:
Just used a bigger device. Now all steps are green.

Reply:
we probably don't need scrollToEnd in onContentSizeChange (sounds like a weird behavior), so I guess onContentSizeChange can be removed?

Reply:
iOS depends on it at the moment. Also it is very useful for newcomers trying the example app on small devices. Before this change if you tap "Show picker" nothing could happen because the picker shows up at the bottom outside of the viewport.
Now, if the picker gets rendered or changed, content Size Changes and the app shows the rendered picker because it scrolls down.
This could also be done not `onContentSizeChange` of the scroll view, but after the picker rendered. Then we would need to wrap the picker component in another component firing an callback which lets the scollView scroll.

I thought this might be to much work for an example app. The solution would be more descriptive than reacting to `onContentSizeChange`.

I will install the example app into an small android simulator and test if the behaviour with scroll to end also works good for android. If not i will limit it to iOS.

Please tell me if the solution like now is ok, or I should bind the scrollToDatepicker more directly to changing the date picker visibility or props.